src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    private Map<String, Integer> fruitMap = new HashMap<>();
+
+    public Map<String, Integer> getFruitMap() {
+        return fruitMap;
+    }
+}
```

Comment:
```suggestion
public class Storage {
    public static Map<String, Integer> fruits = new HashMap<>();
}
```

Reply:
i am not sure about this comment :) but I have fixed it

Reply:
[ERROR] /Users/oleksii/Documents/GitHub/jv-fruit-shop/src/main/java/core/basesyntax/db/Storage.java:7:40: Variable 'fruitMap' must be private and have accessor methods. [VisibilityModifier]